float64 position
---
int8 success
